pkg/pureutils/purefa_rest2x.go

```diff
@@ -75,6 +75,35 @@ func CreatePodinFA(faClient *flasharray.Client, podName string) (*[]flasharray.P
 	return podinfo, nil
 }
 
+// ListAllInterfaces returns list of all Available Interfaces present in FA
+func ListAllInterfaces(faClient *flasharray.Client) ([]flasharray.NetworkInterfaceResponse, error) {
+	interfaces, err := faClient.Network.ListNetworkInterfaces()
+	if err != nil {
+		return nil, err
+	}
+	return interfaces, nil
+}
+
+// EnableInterfaceOnFA enables the given interface on FA
+func EnableInterfaceOnFA(faClient *flasharray.Client, iface string) ([]flasharray.NetworkInterface, error) {
```

Comment:
[nit] same here, we can probably combine Enable/Disable to "SetInterfaceEnabled" as they're nearly identical